app/src/main/java/com/duckduckgo/app/di/NetworkModule.kt

```diff
@@ -89,12 +92,13 @@ class NetworkModule {
 
     @Provides
     @Singleton
-    @Named("pixel")
-    fun pixelRetrofit(@Named("pixel") okHttpClient: OkHttpClient): Retrofit {
+    @Named("nonCaching")
+    fun nonCachingRetrofit(@Named("nonCaching") okHttpClient: OkHttpClient, moshi: Moshi): Retrofit {
         return Retrofit.Builder()
-            .baseUrl(Url.PIXEL)
+            .baseUrl(Url.API)
```

Comment:
ℹ️ `baseUrl` is a required parameter. I've used `Url.API` as hopefully a sensible default, but each service can (and does) override this by providing the full URL it needs.